src/workload.rs

```diff
@@ -254,11 +254,20 @@ impl WorkloadManager {
             None
         };
         let local_workloads = workloads.clone();
-        let local_client = config.local_xds_path.map(|path| LocalClient {
-            path,
-            workloads: local_workloads,
-            block_ready: awaiting_ready,
-        });
+        let local_xds_path = config.local_xds_path.clone();
```

Comment:
why does this change? the logic seems simpler before

Reply:
Hi @howardjohn,  there is an error for string not implement `Copy` trait, so I clone the `local_xds_path`.
I rebased the code and change it to borrow the `local_xds_path` from Arc config for elimination of such complication code.